src/HTTP/Response.php

```diff
@@ -127,6 +127,20 @@ public function has_header(string $name): bool;
      */
     public function get_header(string $name): array;
 
+    /**
```

Comment:
Again, not sure if we should duplicate the comments, rather than using `{@inheritdoc}`